connectors/es/sink.py

```diff
@@ -356,33 +385,48 @@ async def _run(self):
             batch_num = 0
 
             while True:
+                force_flush = False
```

Comment:
I think we don't beed to set `force_flush = True` here, instead we want to only set this after `if isinstance(doc, ForceFlushSignal):` evaluates to True?

I just noticed we set `force_flush = False` in two places one after another.

Reply:
Sorry I don't get it - it's False here and it's only in one place?

Reply:
Yeah ... sorry it might have been late when I reviewed and I misread this as `force_flush = True`  🤦